qa/rpc-tests/mergetoaddress_helper.py

```diff
@@ -129,9 +130,9 @@ def run_test(self, test):
             "Amount out of range",
             lambda: test.nodes[0].z_mergetoaddress(self.any_zaddr_or_utxo, myzaddr, Decimal('21000000.00000001')))
 
-        # Merging will fail because fee is larger than sum of UTXOs
+        # Merging will fail because fee is larger than `-maxtxfee`
         assert_mergetoaddress_exception(
-            "Insufficient funds, have 50.00, which is less than miners fee 999.00",
+            "Fee (999.00 ZEC) is greater than the maximum fee allowed by this instance (0.10 ZEC). Run zcashd with `-maxtxfee` to adjust this limit.",
```

Comment:
This now tests a different case. I think the original one is still testable, but I was having trouble putting together the conditions that lead to it.

There are also a couple other tests that now test the wrong thing and still need to be updated. (E.g., things failing because they try to send to Sprout, rather than because of the condition they intend to test.)